examples/model_selection/plot_randomized_search.py

```diff
@@ -55,7 +56,7 @@ def report(results, n_top=3):
 param_dist = {"max_depth": [3, None],
               "max_features": sp_randint(1, 11),
               "min_samples_split": sp_randint(2, 11),
-              "min_samples_leaf": sp_randint(1, 11),
+              "min_samples_leaf": loguniform(0, 1, base=10),
```

Comment:
Surely min_samples_leaf does not accept a float with the same meaning as an int...?